specification/vi/resource-manager/readme.md

```diff
@@ -52,7 +52,7 @@ tag: package-2021-11-10-preview
 These settings apply only when `--tag=2021-11-10-preview` is specified on the command line.
 
 ```yaml $(tag) == 'package-2021-11-10-preview'
-version: 2021-11-10-preview
+version_with_hyphens: 2021-11-10-preview
 version-with-underscores: 2021_11_10_preview
 input-file:
   - Microsoft.VideoIndexer/preview/2021-11-10-preview/vi.json
```

Comment:
The deduplication flag here is hiding the problems, I suggest we remove this and put it individually in every SDK's configuration

Reply:
videoindexer migrate in https://github.com/Azure/azure-rest-api-specs/pull/17037